libs/vkd3d/vkd3d_shaders.h

```diff
@@ -19,778 +19,28 @@
-    0x00000002,0x00060034,0x00000024,0x0000003e,0x000000aa,0x00000022,0x0000003d,0x00090019,
-    0x00000041,0x00000012,0x00000001,0x00000000,0x00000001,0x00000001,0x00000001,0x00000000,
-    0x00040020,0x00000042,0x00000000,0x00000041,0x0004003b,0x00000042,0x00000043,0x00000000,
-    0x0004003b,0x0000001b,0x00000046,0x00000001,0x00040020,0x0000004a,0x00000003,0x00000012,
-    0x0004003b,0x0000004a,0x0000004b,0x00000003,0x0004003b,0x0000004a,0x0000004c,0x00000003,
-    0x00050036,0x00000002,0x00000004,0x00000000,0x00000003,0x000200f8,0x00000005,0x0004003b,
-    0x00000008,0x00000009,0x00000007,0x0004003b,0x00000028,0x00000029,0x00000007,0x00050041,
-    0x0000000f,0x00000010,0x0000000d,0x0000000e,0x0004003d,0x0000000a,0x00000011,0x00000010,
-    0x0004003d,0x00000013,0x00000017,0x00000015,0x0007004f,0x00000016,0x00000018,0x00000017,
-    0x00000017,0x00000000,0x00000001,0x0004006e,0x0000000a,0x00000019,0x00000018,0x00050080,
-    0x0000000a,0x0000001a,0x00000011,0x00000019,0x0004003d,0x00000006,0x0000001d,0x0000001c,
-    0x00050051,0x00000006,0x0000001e,0x0000001a,0x00000000,0x00050051,0x00000006,0x0000001f,
-    0x0000001a,0x00000001,0x00060050,0x00000007,0x00000020,0x0000001e,0x0000001f,0x0000001d,
-    0x0003003e,0x00000009,0x00000020,0x000300f7,0x00000027,0x00000000,0x000400fa,0x00000025,
-    0x00000026,0x00000027,0x000200f8,0x00000026,0x0004003d,0x0000002a,0x0000002d,0x0000002c,
-    0x0004003d,0x00000007,0x0000002e,0x00000009,0x0007004f,0x0000000a,0x0000002f,0x0000002e,
-    0x0000002e,0x00000000,0x00000002,0x0007005f,0x00000013,0x00000030,0x0000002d,0x0000002f,
-    0x00000002,0x0000000e,0x00050051,0x00000012,0x00000031,0x00000030,0x00000000,0x0003003e,
-    0x00000029,0x00000031,0x000200f9,0x00000027,0x000200f8,0x00000027,0x000300f7,0x00000035,
-    0x00000000,0x000400fa,0x00000033,0x00000034,0x00000035,0x000200f8,0x00000034,0x0004003d,
-    0x00000036,0x00000039,0x00000038,0x0004003d,0x00000007,0x0000003a,0x00000009,0x0007005f,
-    0x00000013,0x0000003b,0x00000039,0x0000003a,0x00000002,0x0000000e,0x00050051,0x00000012,
-    0x0000003c,0x0000003b,0x00000000,0x0003003e,0x00000029,0x0000003c,0x000200f9,0x00000035,
-    0x000200f8,0x00000035,0x000300f7,0x00000040,0x00000000,0x000400fa,0x0000003e,0x0000003f,
-    0x00000040,0x000200f8,0x0000003f,0x0004003d,0x00000041,0x00000044,0x00000043,0x0004003d,
-    0x00000007,0x00000045,0x00000009,0x0004003d,0x00000006,0x00000047,0x00000046,0x0007005f,
-    0x00000013,0x00000048,0x00000044,0x00000045,0x00000040,0x00000047,0x00050051,0x00000012,
-    0x00000049,0x00000048,0x00000000,0x0003003e,0x00000029,0x00000049,0x000200f9,0x00000040,
-    0x000200f8,0x00000040,0x0004003d,0x00000012,0x0000004d,0x00000029,0x0003003e,0x0000004c,
-    0x0000004d,0x0003003e,0x0000004b,0x0000004d,0x000100fd,0x00010038
-};
+typedef enum VKD3D_META_COPY_MODE
+{
+    VKD3D_META_COPY_MODE_1D = 0,
+    VKD3D_META_COPY_MODE_2D = 1,
+    VKD3D_META_COPY_MODE_MS = 2,
+} VKD3D_META_COPY_MODE;
```

Comment:
now I don't mind making this an enum, but it should be consistent with the rest of the code, i.e.
```
enum vkd3d_meta_copy_mode
{
...
```
with no typedef.

Reply:
Can do.

Reply:
Resolved.